### PR TITLE
Hotfix: Add third kwarg back to _get for backwards compatibility

### PR DESCRIPTION
**Pull request by fwkoch**

### CODE REVIEW

File: properties/base.py [hunk 1]
```diff
@@ -155,10 +155,15 @@ def __init__(self, **kwargs):
                                'or attribute'.format(key))
             setattr(self, key, kwargs[key])
 
-    def _get(self, name):
+    def _get(self, name, default=None):
         if name in self._backend:
             return self._backend[name]
 
+        if default is utils.undefined:
```
Comment (ThomasMatern) [inline properties/base.py]: This module uses utils.undefined as well as basic.undefined. It's the same thing but consistency would be great.

File: properties/base.py [hunk 2]
```diff
@@ -155,10 +155,15 @@ def __init__(self, **kwargs):
...
+            return None
+        if default is not None:
+            return default
+
```
Comment (ThomasMatern) [inline properties/base.py]: Shouldn't this set self._backend[name] as below to set a fixed default value?

File: properties/base.py (hunk 1 above)
Comment (fwkoch) [inline properties/base.py]: Thanks, yes it should be `utils.undefined` everywhere.

File: properties/base.py (hunk 2 above)
Comment (fwkoch) [inline properties/base.py]: Yes, you are correct. This was intended as a quick fix for some problems other users were having with backwards incompatibility. However, they have already modified their code accordingly, so I don't think this PR will be merged.

File: properties/base.py (hunk 1 above)
Comment (fwkoch) [inline properties/base.py]: Just to tidy up loose ends, this was fixed here: https://github.com/3ptscience/properties/commit/7d1e4d662d50d55303e908ba5e30316d9a338862

**Pull request closed without merging**